snapcraft/plugins/python.py

```diff
@@ -277,17 +287,44 @@ def __init__(self, *, exec_func, runnable, package_dir, env,
         if dependency_links:
             self._extra_pip_args.append('--process-dependency-links')
 
+    def list(self, exec_func=None):
```

Comment:
Ah, I see. I didn't notice this was inside the Pip class. It is good, I'm sorry.